lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -11,9 +11,12 @@ defmodule ExUnit.Case do
 
   When used, it accepts the following options:
 
-    * :async - configure this specific test case to able to run in parallel
+    * `:async` - configure this specific test case to able to run in parallel
```

Comment:
Not sure, but yeah let's mention in the capturing modules 👍Closing this, will open another PR later!